docs/README.md

```diff
@@ -61,7 +61,7 @@ See also https://issues.apache.org/jira/browse/SPARK-35375.
 -->
 
 ```sh
-$ sudo pip install 'sphinx<3.1.0' mkdocs numpy pydata_sphinx_theme ipython nbsphinx numpydoc sphinx-plotly-directive 'jinja2<3.0.0'
+$ sudo pip install 'sphinx<3.1.0' mkdocs numpy pydata_sphinx_theme ipython nbsphinx numpydoc sphinx-plotly-directive pyarrow 'jinja2<3.0.0' 'markupsafe==2.0.1' 'docutils<0.18.0'
```

Comment:
Maybe let's just fix this to `sudo pip install dev/requirements.txt`

Reply:
I was thinking about that..

But the requirements.txt makes you download many more packages which aren't specific to the documentation. I feel like it is good as is, we could consider adding a section/pointing to the requirements.txt for an alternative

Reply:
That's fine. You can just mention like "`dev/requirements.txt` includes all packages for developement so this is the easiest way to install them". (feel free to rephrase it)